stdlib/database/h2/src/main/ballerina/h2/Module.md

```diff
@@ -4,11 +4,85 @@ This module provides the functionality required to access and manipulate data st
 
 ### Client
 
-To access a database, you must first create a `client` object. Create a client of the H2 Client type (i.e., `h2:Client`) and provide the necessary connection parameters. This will create a pool of connections to the given H2 database. A sample for creating a H2 client can be found below.
+To access a database, you must first create a `client` object. A sample for creating an H2 client can be
+found below.
+
+### Connection pool handling
+
+There are 3 possible scenarios for connection pool handling.
+
+1. Global, shareable default connection pool
```

Comment:
```suggestion
## Creating a global shareable connection pool with the default configurations
```
Shall we make this a subtopic?

Reply:
Global pool cannot be created by the user so we can't rephrase this way